### DISCUSSION TITLE
GraphScope GAE is working on ARM Kylin OS v10

### DISCUSSION THREAD

**Discussion by TerryWang1208** [alibaba/GraphScope#2501] [Show and tell]
Per request of our client, we need to run GraphScope on ARM Kylin V10, but there has no images or python wheels from the community. So we built the source code and addressed all kinds of dependencies. Finally we successfully packaged all wheels which can be installed repeatedly on the os of ARM Kylin V10.

Just let you know this possibility and glad to share our experience if needed. And look forward to the official support from GraphScope team on ARM Kylin V10. There are lots of big clients who are being forced to use this OS and this is a trending for the coming 1-2 years.

**Comment by sighingnow** [comment]
We have delivered prilimary arm support as well, at least our base image (i.e., `graphscope/graphscope-dev`) already has ARM support. Maybe @siyuan0322 can tell more about that.

As M1-chip based MacOS become popular, we believe ARM support is important for us, too. 

> Just let you know this possibility and glad to share our experience if needed.

It would be great to have such collaboration!

**Comment by siyuan0322** [comment]
> And look forward to the official support from GraphScope team on ARM Kylin V10

We are planning to extend our support to ARM-based platforms for all our products. Currently, we have our development images available on a dual platform. However, we aim to include ARM support in all our images with every release, and set up Continuous Integration to ensure thorough testing.

> Just let you know this possibility and glad to share our experience if needed.

That would be great. We really appreciate it!

**Comment by wuyueandrew** [comment]
> Per request of our client, we need to run GraphScope on ARM Kylin V10, but there has no images or python wheels from the community. So we built the source code and addressed all kinds of dependencies. Finally we successfully packaged all wheels which can be installed repeatedly on the os of ARM Kylin V10.
> 
> Just let you know this possibility and glad to share our experience if needed. And look forward to the official support from GraphScope team on ARM Kylin V10. There are lots of big clients who are being forced to use this OS and this is a trending for the coming 1-2 years.

im very insteresting in deploying gs on arm + kylin ， would u like to share more detail like ur k8s distribution.
